.tx/config

```diff
@@ -2,8 +2,8 @@
 host = https://www.transifex.com
 lang_map = bg_BG: bg, cs_CZ: cs, fi_FI: fi, hu_HU: hu, nb_NO: nb, sk_SK: sk, th_TH: th, ja_JP: ja
 
-[nextcloud.fileslibreofficeedit]
```

Comment:
That is actually the appid so I think it's good without the _

Reply:
In case it doesn't have to be the app id I'm fine. I just think it would be easier in terms of pattern and uniqueness if it is the app id